src/index.ts

```diff
@@ -213,65 +228,63 @@ function createRdfEmitter(program: Program) {
                 }
               }
 
-              writer_constraints.addQuad(
-                nameNodeShacl,
-                nn("sh:or"),
-                writer_constraints.list(arr)
+              constraintQuads.push(
+                quad(
+                  nameNodeShacl,
+                  nn("sh:or"),
+                  writer.list(arr) as any // error in n3 typing, this is supported

```

Comment:
I verified that this works despite the type error. The types are just wrong, unfortunately, so we have to cast to any to get around it.

Reply:
Right! I was really struggling with this part before.